lambdas/service/app.py

```diff
@@ -913,6 +963,14 @@ def repository_search(entity_type: str, item_id: str):
         logic. For example, `{"donorCount": {"within": [[1,5], [5,10]]}}`
         selects entities whose donor organism count falls within both
         ranges, i.e., is exactly 5.
+
+        When filtering on `organismAge` facet, the user must specify a
+        dictionary with `value` and `unit` fields. For example,
+        `{"organismAge": {"is": [{'value': '20', 'unit': 'year'}]}}`, this
+        returns a response for entries where organismAge is `20 year`. For
+        filtering on response entries where `organismAge` is unknown,
+        `value` and `unit` must be set to None. For instance
+        `{"organismAge": {"is": [{'value': None, 'unit': None}]}}`.
```

Comment:
@danielsotirhos I've modified the description from its original form, how may I improve it!?

Reply:
With the two preceding paragraphs of the `filters_param_spec` in mind, here's what I came up with:

The `organismAge` facet is unique in that it contains two property keys `value` and `unit`. For example, `{"organismAge": {"is": [{'value': '20', 'unit': 'year'}]}}`. Both keys are required however the value `null` can be used to specify a non-value.  For example, `{"organismAge": {"is": [{'value': null, 'unit': null}]}}` selects entities with no `organismAge` value.

Reply:
Looks good: s/unique/special and s/non-/unspecified or absent /